code/go/0chain.net/smartcontract/minersc/global_node.go

```diff
@@ -339,90 +339,138 @@ func (gn *GlobalNode) Get(key Setting) (interface{}, error) {
 
 // The prevMagicBlock from the global node (saved on previous VC) or LFMB of
 // the balances if missing (genesis case);
-func (gn *GlobalNode) prevMagicBlock(balances cstate.StateContextI) (
-	pmb *block.MagicBlock) {
+func (gn *GlobalNode) prevMagicBlock(balances cstate.StateContextI) (pmb *block.MagicBlock, err error) {
```

Comment:
I think since we have a single magic block now it won't break the MPT consistency, right?

Reply:
yes, it won't break the MPT consistency.